### ISSUE TITLE
Build Failed with RN 0.58.4

### ISSUE THREAD

**Issue by sreejitrozario** [andpor/react-native-sqlite-storage#327]
Build is failing. It is just a simple starter app.
Build is failing once the gradle file is updated (`compile project(':react-native-sqlite-storage')` is added to the dependencies in  **/app/build.gradle** )

![screenshot from 2019-02-19 16-23-45](https://user-images.githubusercontent.com/23395416/53010403-da46f200-3463-11e9-87d7-1cbe6a3cb4d6.png)

**Comment by codemeall** [comment]
@sreejitrozario did you resolve the issue? getting the same issue.

**Comment by William8Cabral** [comment]
same here

**Comment by DiegoBittencourtOfficial** [comment]
Same here, waiting for an official solution, nothing that I did worked...

**Comment by codemeall** [comment]
Hi all, updating gradle to latest version resolved my issue. Switch to android studio and update gradle and BOOM it works :)

**Comment by DiegoBittencourtOfficial** [comment]
> Hi all, updating gradle to latest version resolved my issue. Switch to android studio and update gradle and BOOM it works :)

Can you post some steps here to help us? =)

**Comment by DiegoBittencourtOfficial** [comment]
@codemeall  worked, thanks!!!

**Comment by braksator** [comment]
try

`    implementation project(':react-native-sqlite-storage')`

**Comment by William8Cabral** [comment]
@codemeall worked, ty

**Comment by dmuensterer** [comment]
@William8Cabral @DiegoBittencourtOfficial @codemeall Which Gradle version are you using? Can't get it to work somehow.

**Comment by giomodiogo** [comment]
Hello everyone.

I solved my problem as follows:

- Open the project in Android Studio and remove the code below build.gradle (project):

```
task wrapper(type: Wrapper) {
     gradleVersion = '4.7'
     distributionUrl = distributionUrl.replace("bin", "all")
}

```

Save and wait for the build, or click sync now.

- Open the file: gradle-wrapper.properties and change the version to gradle to 4.10.1:

`distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip`

Save and wait to download the dependencies.

- Go back to the build.gradle file and re-add the code, with the new version:

```
task wrapper(type: Wrapper) {
     gradleVersion = '4.10.1'
     distributionUrl = distributionUrl.replace("bin", "all")
}

```

Wait for the build and have fun.

Thank you.

**Comment by bogrodev** [comment]
Thank you, you saved me

**Comment by andpor** [comment]
I marged the [PR](https://github.com/andpor/react-native-sqlite-storage/pull/324) with the fix and released 3.3.10
